terraform/modules/bulk_upload/access_log_bucket.tf

```diff
@@ -0,0 +1,101 @@
+#tfsec:ignore:aws-s3-enable-bucket-encryption: default bucket encryption is sufficient
+#tfsec:ignore:aws-s3-enable-bucket-logging: access log bucket doesn't need access logging itself
+#tfsec:ignore:aws-s3-encryption-customer-key: default encryption is sufficient
+resource "aws_s3_bucket" "bulk_upload_access_logs" {

```

Comment:
Why name the resource `_access_logs` but the bucket itself just `_logs`?

Reply:
`_logs` follows the convention we set for the backend log bucket names, but I felt `access_logs` made things clearer in the terraform itself since we're doing the implementation ourselves here (rather than an external module as per the backend)